demisto_sdk/commands/common/content/objects/abstract_objects/general_object.py

```diff
@@ -86,12 +95,10 @@ def dump(self, dest_dir: Optional[Union[Path, str]] = None) -> List[Path]:
         Returns:
             List[Path]: List of path created in given directory.
 
-        TODO:
-            1. Implement dump of modified object.
         """
-        dest_file = self._create_target_dump_dir(dest_dir) / self.normalize_file_name()
 
-        return [copyfile(src=self.path, dst=dest_file)]
+        dest_file = self._create_target_dump_dir(dest_dir) / self.normalize_file_name()
+        return [copyfile(src=self._path, dst=dest_file)]
```

Comment:
not sure what is the change here

Reply:
Removed the 'TODO' as dumping modified object is now supported